README.md

```diff
@@ -1,7 +1,7 @@
 About bottleneck
 ================
 
-Home: https://github.com/kwgoodman/bottleneck
+Home: https://github.com/pydata/bottleneck
```

Comment:
You have to change it in the `meta.yaml` and then re-render. The README is auto-generated.

Reply:
Thanks - done and updated a few other things in there